pkg/config/toml/config.go

```diff
@@ -628,9 +628,12 @@ func (a *Automation) setFrom(f *Automation) {
 }
 
 type Workflow struct {
-	FromAddress      *types.EIP55Address `toml:",omitempty"`
-	ForwarderAddress *types.EIP55Address `toml:",omitempty"`
-	GasLimitDefault  *uint64
+	FromAddress       *types.EIP55Address `toml:",omitempty"`
+	ForwarderAddress  *types.EIP55Address `toml:",omitempty"`
+	GasLimitDefault   *uint64
+	TxAcceptanceState *commontypes.TransactionStatus
+	PollPeriod        *commonconfig.Duration
+	AcceptanceTimeout *commonconfig.Duration
```

Comment:
What are these configs? scoped to a workflow?

They don't seem to be used in this PR.

Reply:
Aha, I see, they are used in the core PR: https://github.com/smartcontractkit/chainlink/pull/17216/files#diff-9b55aa19c69723ca59deb033266f910479defbc34e400b37e1c42a8897c3a373R127

Well, 2-3 things here:

- Configs are inconsistent (e.g., AcceptanceTimeout vs ConfirmerTimeout)
- Configs should be namespaced and scoped - this is not a Workflow config but a WT capability config (should be expressed and nested that way in toml)
- Configs should be composable (just import framework defined configs and use those, don't redefine and rename)

Reply:
Fixed first bullet point, scoped out ticket for following two: https://smartcontract-it.atlassian.net/browse/PLEX-1536